mock/src/main/java/com/tngtech/keycloakmock/impl/TokenGenerator.java

```diff
@@ -56,12 +55,41 @@ public String getToken(
             .claim("scope", tokenConfig.getScope())
             .claim("typ", "Bearer")
             .claim("azp", tokenConfig.getAuthorizedParty());
+    Optional<UserData> generatedUserData;
```

Comment:
Could use ternary

Reply:
Hm, yes. But it's not significantly shorter, and the google formatting looks weird:

```
    Optional<UserData> generatedUserData =
        tokenConfig.isGenerateUserDataFromSubject() ? Optional.of(
            UserData.fromUsernameAndHostname(
                tokenConfig.getSubject(), requestConfiguration.getHostname())) : Optional.empty();
```